cli/cli.go

```diff
@@ -1085,14 +1085,37 @@ func argsCountError(ctx *cli.Context) error {
 	return fmt.Errorf(
 		ArgumentCountError,
 		ctx.App.Name,
-		ctx.Command.Name,
+		commandFullName(ctx, os.Args),
 	)
 }
 
 func argsParseError(ctx *cli.Context) error {
 	return fmt.Errorf(
 		ArgumentParsingError,
 		ctx.App.Name,
-		ctx.Command.Name,
+		commandFullName(ctx, os.Args),
 	)
 }
+
+// because ctx.Command.FullName() doesn't work: https://github.com/urfave/cli/issues/1859
+func commandFullName(ctx *cli.Context, args []string) string {
```

Comment:
Since this needs whole context, maybe it would make sense to modify template to not use `ctx.App.Name` separately and add it here?

Reply:
Now the function returns `appName + args`, also modified the message to accept one `%s`.